--- jpilot-0.99.5/configure.in.orig	2003-04-20 15:27:15.000000000 +0200
+++ jpilot-0.99.5/configure.in	2003-04-20 15:29:43.000000000 +0200
@@ -468,7 +468,7 @@
 AM_CONDITIONAL(MAKE_EXPENSE, test x$plugin_support = xyes)
 AM_CONDITIONAL(MAKE_SYNCTIME, test x$plugin_support = xyes)
 
-AC_OUTPUT(Makefile docs/Makefile Expense/Makefile KeyRing/Makefile intl/Makefile dialer/Makefile po/Makefile.in SyncTime/Makefile jpilot.spec SlackBuild)
+AC_OUTPUT(Makefile docs/Makefile Expense/Makefile KeyRing/Makefile intl/Makefile dialer/Makefile SyncTime/Makefile po/Makefile.in jpilot.spec SlackBuild)
 
 chmod +x SlackBuild
 
